docs/resources/css_logstash_custom_certificate.md

```diff
@@ -0,0 +1,82 @@
+---
+subcategory: "Cloud Search Service (CSS)"
+---
+
+# huaweicloud_css_logstash_custom_certificate
+
+Manages CSS logstash cluster custom certificate resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" {}
+variable "bucket_name" {}
+variable "cert_object" {}
+
+resource "huaweicloud_css_logstash_custom_certificate" "test"  {
+  cluster_id  = var.cluster_id
+  bucket_name = var.bucket_name
+  cert_object = var.cert_object
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this creates a new resource.
+
+* `cluster_id` - (Required, String, ForceNew) Specifies ID of the CSS logstash cluster.
+  Changing this creates a new resource.
+
+* `bucket_name` - (Required, String, ForceNew) Specifies the OBS bucket name where the certificate file is stored.
+  Changing this creates a new resource.
+
+* `cert_object` - (Required, String, ForceNew) Specifies the certificate file path to upload in the OBS bucket.
+  The certificate name ranges from **4** to **32** digits and must start with a letter
+  (.cer|.crt|.rsa|.jks|.pem|.p10|.pfx|.p12|.csr|.der|.keystore) at the end, it can contain letters, numbers,
```

Comment:
```suggestion
  The certificate name ranges from **4** to **32** digits and must start with a letter
  (.cer|.crt|.rsa|.jks|.pem|.p10|.pfx|.p12|.csr|.der|.keystore at the end), it can contain letters, numbers,
```